packages/webapp/src/__tests__/OrderRemoveModal.test.ts

```
@@ -28,12 +25,13 @@ describe('OrderRemoveModal', () => {
 	} as unknown as OrderRemoveModalProps;
 
-	beforeEach(() => {
-		vi.clearAllMocks();
+	beforeEach(async () => {
 		transactionStore.reset();
+		console.log('getRemoveOrderCalldata !!!', getRemoveOrderCalldata);
+	(getRemoveOrderCalldata as Mock).mockResolvedValue('0x123');
 	});
 
-	it('handles transaction correctly', async () => {
+	it.only('handles transaction correctly', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)